domains/authorization/Proxy.IntegrationTests/ProxyBaseIntegrationTests.cs

```diff
@@ -11,21 +9,21 @@ namespace Proxy.IntegrationTests;
 
 public class ProxyBaseIntegrationTests(ProxyIntegrationTestFixture fixture) : IClassFixture<ProxyIntegrationTestFixture>
 {
-    private HttpClient CreateClientWithOrgIds(List<string> orgIds)
+    private HttpClient CreateClientWithOrgIds(Guid orgId, List<string> orgIds)
     {
-        return fixture.Factory.CreateAuthenticatedClient(orgIds: orgIds);
+        return fixture.Factory.CreateAuthenticatedClient(orgId: orgId.ToString(), orgIds: orgIds);
     }
 
     [Fact]
     public async Task Proxy_Forwards_Request_To_Downstream_With_Version_Header()
     {
-
         fixture.WalletWireMockServer
             .Given(Request.Create().WithPath("/wallet-api/v1/certificates").UsingGet())
             .RespondWith(Response.Create().WithStatusCode(200));
 
+        var orgId = Guid.NewGuid();
         var orgIds = new List<string> { Guid.NewGuid().ToString(), Guid.NewGuid().ToString() };
-        var client = CreateClientWithOrgIds(orgIds);
+        var client = CreateClientWithOrgIds(orgId, orgIds);
```

Comment:
hvad er forskellen med orgId og OrgIds

Reply:
Two different claims in the token, _org_id_ contains the users own organization, the one the user is affiliated with. The _org_ids_ claim contains ids on all organization the users organization has been given consent to.